helper-cli/src/main/kotlin/common/Utils.kt

```diff
@@ -200,26 +200,33 @@ internal fun OrtResult.fetchScannedSources(id: Identifier): File {
 internal fun OrtResult.getProcessedCopyrightStatements(
     omitExcluded: Boolean = true,
     copyrightGarbage: Set<String> = emptySet()
-): Map<Identifier, Map<String, Set<String>>> {
-    val result = mutableMapOf<Identifier, MutableMap<String, MutableSet<String>>>()
+): Map<Identifier, Map<String, Map<String, Set<String>>>> {
```

Comment:
That return type is getting a bit out of hand, please adapt at least the documentation of the function.
Do you think it would better to use `Map<Identifier, Map<String, CopyrightStatementProcessor.Result>>` instead? It would make it easier to understand the content of the object and the caller could use `Result.toMutableMap()` if they only want the statements, which, if I understood that correctly, would be the same as the key set of the inner most map.
Btw, off-topic, but I think we should rename `Result.toMutableMap()` to something more speaking like `Result.allCopyrightStatements()`.

Reply:
> Btw, off-topic, but I think we should rename Result.toMutableMap() to something more speaking like Result.allCopyrightStatements().

I have already a commit laying around for that.

I would love to keep the return type as is as I plan to add a parameter to the ListCopyrightsCommand to group by package and license optionally.

Reply:
> I have already a commit laying around for that.

Nice, looking forward to it.
 
> I would love to keep the return type as is as I plan to add a parameter to the ListCopyrightsCommand to group by package and license optionally.

I just had the impression that the inner most map is basically the same as the `Result` object, but we can discuss this next week.